.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+sudo: required
+dist: trusty
+matrix:
+  include:
+    - python: "3.6"
+      env:
+        - INTERPRETER=python3
+    - python: "pypy3.5"
+      env:
+        - INTERPRETER=pypy3
+    - python: "3.4"
+      env:
+        - INTERPRETER=python3
+before_script:
+    $INTERPRETER -m pip install -r atest/requirements.txt
+script:
+  - echo $INTERPRETER
+  - $INTERPRETER --version
+  - $INTERPRETER utest/run.py
+  - $INTERPRETER atest/run.py $INTERPRETER --exclude no-ci atest/robot
```

Comment:
Idea: comment this out for now. First get unittests running on jenkins, then later add acceptance tests. Also, please add a badge (like `[![Build Status](https://travis-ci.org/RustPython/RustPython.svg?branch=master)](https://travis-ci.org/RustPython/RustPython)`)